CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,120 @@
+# Code of Conduct
+
+([Français](#Code-de-conduite))
+
+Contributors are expected to follow the Code of Conduct, and those working within Government are also expected to follow the Values and Ethics Code for the Public Sector
+
+## Values and Ethics Code for the Public Sector
+
+The [Values and Ethics Code for the Public Sector](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=25049)
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment include:
+
+* Using welcoming and inclusive language
```

Comment:
This is a little vague, I think this is covered under being respectful and showing empathy below. It could also be changed to something like "Being friendly and inclusive of others" or something along those lines.

Reply:
Agreed...will update with the following example: 
"E.g. comments must not contain profanity or discrimination on the basis of, for example, race, national or ethnic origin, colour, religion, sex, age, mental or physical disability, or sexual orientation;"